website/src/components/blog/BlogPost.tsx

```diff
@@ -46,23 +46,39 @@ export const BlogPost: React.FunctionComponent<Props> = ({
                         post.frontmatter.title
                     )}
                 </h1>
-                {post.frontmatter.author && post.frontmatter.publishDate && (
+                {post.frontmatter.authors?.length && (
                     <p className="blog-post__byline mb-0">
-                        {post.frontmatter.authorUrl ? (
-                            post.frontmatter.authorUrl.includes('http') ? (
-                                <a href={post.frontmatter.authorUrl} target="_blank" rel="nofollow noreferrer">
-                                    {post.frontmatter.author}
-                                </a>
-                            ) : (
-                                <Link to={post.frontmatter.authorUrl}>{post.frontmatter.author}</Link>
+                        {post.frontmatter.authors.map((a, index) => {
```

Comment:
I tested several byline cases and found that your solution is super robust! No unexpected styles here:

One author with link:
<img width="600" alt="Screen Shot 2022-05-17 at 3 33 49 PM" src="https://user-images.githubusercontent.com/60713139/168909347-e1746cf8-207a-47a7-87df-a169c4498f93.png">

Multiple authors with links:
<img width="600" alt="Screen Shot 2022-05-17 at 3 33 44 PM" src="https://user-images.githubusercontent.com/60713139/168909345-0fb0e213-7e5a-4b8a-b4fb-78ed889144ad.png">

One author without a link:
<img width="600" alt="Screen Shot 2022-05-17 at 4 02 11 PM" src="https://user-images.githubusercontent.com/60713139/168909648-41a2c1ac-f44d-4839-ae7d-68c59db151ab.png">

Multiple authors without links:
<img width="600" alt="Screen Shot 2022-05-17 at 3 33 37 PM" src="https://user-images.githubusercontent.com/60713139/168909307-1f035490-4de9-43d8-b196-6579239a5bb7.png">

Reply:
Woohoo 😸